tsconfig.json

```diff
@@ -4,6 +4,13 @@
 		"declaration": false,
 		"emitDecoratorMetadata": true,
 		"experimentalDecorators": true,
+		"lib": [
+			"dom",
+			"es5",
+			"es2015.iterable",
+			"es2015.symbol",
+			"es2015.symbol.wellknown"
+		],
```

Comment:
It wasn't/isn't needed in the `shim` one, the `core` PR also did not include the `intern` types in the `tsconfig.json`.